src/ui/styles/separator.scss

```diff
@@ -0,0 +1,6 @@
+.separator-line {
```

Comment:
I can see this is for the template and what matches figma - I think this is just a figma thing and not actually required, we have other summary lists with a heading and they do not have a separator line, Doesn't look to be a GOV design pattern

Reply:
done